src/nft/components/details/detailsV2/DataPage.tsx

```diff
@@ -0,0 +1,55 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+
+import { DataPageDescription } from './DataPageDescription'
+import { DataPageHeader } from './DataPageHeader'
+import { DataPageTable } from './DataPageTable'
+import { DataPageTraits } from './DataPageTraits'
+
+const DataPageContainer = styled(Column)`
+  padding: 24px 64px;
+  height: 100vh;
+  width: 100%;
+  gap: 36px;
+  max-width: 1200px;
```

Comment:
this is a const only for the Data Page as used in the Figma

Reply:
Is it referenced somewhere else in styles? Worth extracting as a theme value? Is there any other container width used by other pages? Similar? Maybe we can unify?

Reply:
I dug and found that this max-width is used in a couple of locations in NFTs and Token Explore. Token Explore has set up their own set of breakpoints in `src/components/Tokens/constants.ts` including `MAX_WIDTH_MEDIA_BREAKPOINT = '1200px'`. I've set a `maxWidth` in the theme file and implemented it where used in NFT pages. To avoid scope creep and introducing regressions I did not alter the Token explore code.